src/standalone/Vegetation/PlantHydraulics.jl

```diff
@@ -591,8 +592,8 @@ function root_flux_per_ground_area!(
     model::PlantHydraulicsModel{FT},
     Y::ClimaCore.Fields.FieldVector,
     p::NamedTuple,
-    t::FT,
-) where {FT}
+    t,
+) where {FT <: AbstractFloat}
```

Comment:
I dont know that we need to enforce this here, because FT <: AbstractFloat is part of the definition of AbstractCanopyComponent{FT}: https://github.com/CliMA/ClimaLSM.jl/blob/ad31773009c9309d5e45b083603873a8914be286/src/standalone/Vegetation/component_models.jl#L21

Reply:
Ahh good point, I'll remove all the `<: AbstractFloat`s